package.json

```diff
@@ -35,13 +35,10 @@
   "dependencies": {
     "@babel/core": "7.2.2",
     "@fortawesome/free-solid-svg-icons": "5.1.0",
-    "@patternfly/patternfly": "2.71.6",
-    "@patternfly/react-charts": "5.3.21",
-    "@patternfly/react-core": "3.158.1",
-    "@patternfly/react-icons": "3.15.17",
-    "@patternfly/react-styles": "3.7.14",
-    "@patternfly/react-table": "2.28.47",
-    "@patternfly/react-tokens": "2.8.14",
```

Comment:
Why were `react-icons`, `react-tokens`, and `react-styles` removed as dependencies?